bindings/cs/MongoDB.Libmongocrypt.Test/BasicTests.cs

```diff
@@ -520,7 +557,8 @@ public void TestLocalKeyCreationStepwise()
                             var binary = req.Message;
                             _output.WriteLine("Key Document: " + binary);
                             var postRequest = binary.ToString();
-                            postRequest.Should().Contain("Host:kms.us-east-1.amazonaws.com");
+                            // TODO: add different hosts handling
+                            postRequest.Should().Contain("Host:kms.us-east-1.amazonaws.com"); // only AWS
```

Comment:
This could also assert that the Host header matches the host name of the `req.Endpoint`.

Reply:
I agree that it could be a good asserting, but as I see, it will require `string` analysis, and since this change is not directly related to this PR I would not do it right now.